lib/src/main/java/ECSE428Project/service/LogoutService.java

```diff
@@ -0,0 +1,47 @@
+package ECSE428Project.service;
+
+import ECSE428Project.dao.AccountRepository;
+import ECSE428Project.model.Account;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+import static org.hibernate.internal.util.collections.ArrayHelper.toList;
+
+@Service
+public class LogoutService {
+
+    @Autowired
+    private AccountRepository accountRepository;
+
+
+    //Do we need to implement a "find current logged in account" method?
```

Comment:
Yeah we should create a getter method that finds an account and checks if it's logged-in, we'll do that during the next sprint